x-pack/metricbeat/docker-compose.yml

```diff
@@ -10,6 +10,7 @@ services:
     volumes:
       - ${PWD}/../..:/go/src/github.com/elastic/beats/
       - /var/run/docker.sock:/var/run/docker.sock
+      - ${HOME}/.docker:/root/.docker:ro
```

Comment:
Fun fact, `PWD` and `HOME` are not available in docker-compose on windows. If we want to be able to run full system tests in the future on windows as well, we will have to do some modifications.

Reply:
yep both env vars impact developers on local laptops that are testing `x-pack/metricbeat`, so there were impacted by `${PWD}` before this PR, and not they are impacted too by `${HOME}`.
Also, any windows developer running Docker configured for windows containers cannot run the tests (windows only allow to install windows containers support or Linux containers support not both).

Reply:
> Also, any windows developer running Docker configured for windows containers cannot run the tests (windows only allow to install windows containers support or Linux containers support not both).

Right. But docker-compose is evaluated from the POV of the host. So it doesn't work with linux containers on a windows host (I tried :) ). But this is not new. Replacing `PWD` with `.` helps btw.

The main impact is not `${PWD}` or `${HOME}` per se. But developers will need to create a `.docker` file/directory. What shall we include there in order to test locally?

Reply:
IIRC `/var/run/docker.sock` does not work on windows too. The compose file only works on Linux right now, add the Docker credendial mounted on the containers will work on any case if the file does not exist will mount nothing. The Oracle test is enabled by an env var disabled by default, so I think it is not an issue to add the Docker folder to the container.